docs/what-can-be-faked.md

```diff
@@ -13,7 +13,7 @@ following entities can be faked:
     * are not sealed,
     * are not static, and
     * have at least one public or protected constructor whose arguments FakeItEasy can construct or obtain
-
+* delegates

```

Comment:
I'd made a comment about bad indentation. My mistake. My context was too narrow. This is great.